frontend/angular-cli.json

```diff
@@ -15,9 +15,14 @@
       "prefix": "app",
       "mobile": false,
       "styles": [
-        "styles.css"
+          "outcobra.scss",
+          "../node_modules/materialize-css/bin/materialize.css",
+          "../node_modules/material-design-icons/iconfont/material-icons.css"
+      ],
+      "scripts": [
+          "../node_modules/jquery/dist/jquery.js",
+          "../node_modules/materialize-css/bin/materialize.js"
```

Comment:
@jmesserli You don't have to add moment there. We just have to add a new dependency in the package.json and ensure that we have the TypeScript types installed and then the angular-cli will automatically bundle moment with all the other stuff.